### PR TITLE
LPS-125579 - Update npm-scripts to 36.8.0

### PR DESCRIPTION
**Pull request by bryceosterhaus**
This is a follow up from https://github.com/liferay-frontend/liferay-portal/pull/617

- Updates npm-scripts to [36.8.0](https://github.com/liferay/liferay-frontend-projects/releases/tag/npm-scripts%2Fv36.8.0)
- Updates gradle plugins to only run the buildCSS and copyCSS tasks if the `build` npm script doesn't exist
- Removes any specified buildCSS and copyCSS tasks from modules that rely on npm-scripts

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
@john-co could you verify these failures? I couldn't seem to reproduce locally.

**Comment by john-co** [comment]
ci:test:bundle

**Comment by john-co** [comment]
Reviewed. Test failures are unrelated. :heavy_check_mark: 
- These FormRules tests are still flaky, I will send a commit to this PR to quarantine them @cvanut
- PagesAdmin test looks like a fluke cc/ @david-gutierrez-mesa page 2 was listed below page 3 for some reason. I'm assuming that the test is expecting the list to be sorted by created date by default?

**Comment by john-co** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
Now just gonna wait for further review before we forward this.

File: modules/apps/adaptive-media/adaptive-media-web/build.gradle [hunk 1]
```diff
@@ -1,8 +1,4 @@
-buildCSS {
-	imports = [
-		new File(npmInstall.nodeModulesDir, "@clayui/css/src/scss")
```
Comment (jbalsas) [inline modules/apps/adaptive-media/adaptive-media-web/build.gradle:3]: This means that `@clayui/css` is added automatically to the list of Sass dependencies and there's no need to add it manually anymore?

**Review by jbalsas — APPROVED** [review]
LGTM @bryceosterhaus! 

This is in need of a rebase and I would forward it to @petershinn if he hasn't approved this yet. Otherwise, tests are green, so I would go for it 😂

File: modules/apps/adaptive-media/adaptive-media-web/build.gradle (hunk 1 above)
Comment (bryceosterhaus) [reply modules/apps/adaptive-media/adaptive-media-web/build.gradle:3]: @jbalsas correct, we have this list of sass dependencies that are automatically added [here](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/npm-tools/packages/npm-scripts/src/presets/standard/index.js#L56-L60).

Ideally we remove bourbon once we get modules to stop referencing it.

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:relevant

**Comment by yichenroy** [comment]
ci:test:sf

**Comment by yichenroy** [comment]
ci:test:sf

**Comment by yichenroy** [comment]
ci:test:sf

**Comment by yichenroy** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
ci:forward

**Comment by jbalsas** [comment]
@bryceosterhaus, went to forward this but @izaera was faster and got https://github.com/brianchandotcom/liferay-portal/pull/97598 merged!

@izaera, could you rebase this and push it to see if we can get this through as well so we won't have a way to know what exactly did we break? 😂

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
Updated and removed my commit for updating npm-scripts, that should be handled by @izaera's update already.

**Comment by john-co** [comment]
ci:test:relevant

**Comment by john-co** [comment]
seems like a random failure? I don't think it's related to the PR so rerunning the tests.

**Comment by john-co** [comment]
cc/ @PablitoBonito

**Comment by john-co** [comment]
I'd say this is issue is unrelated... another java runtime error but looks like it's showing a different error now. I contacted ci channel to take a look.

**Comment by izaera** [comment]
I assume you don't need my help (as per @bryceosterhaus [comment](https://github.com/liferay-frontend/liferay-portal/pull/680#issuecomment-758862073) ). If that's not the case, please ping me again. Thx :+1:

**Comment by izaera** [comment]
For reference. It looks like the CI error was a [bug](https://issues.liferay.com/browse/LRCI-1915).

**Comment by bryceosterhaus** [comment]
Verified with @john-co to manually forward at https://github.com/brianchandotcom/liferay-portal/pull/97691

**Pull request closed without merging**